Lib/test/test_os.py

```diff
@@ -87,6 +88,60 @@ def test_getcwd(self):
         cwd = os.getcwd()
         self.assertIsInstance(cwd, str)
 
+    def test_getcwd_long_path(self):
+        # bpo-37412: On Linux, PATH_MAX is usually around 4096 bytes. On
+        # Windows, MAX_PATH is defined as 260 characters, but the universal
+        # naming convention (UNC) allows longer paths. Internally, the os
```

Comment:
To be clear, UNC paths have the same length limit as DOS drive paths. "\\\\?\\" paths remove the limit because they bypass normalization, but these are not UNC paths.

Reply:
Hum. I rephased the comment in a more generic way.

"On Windows, MAX_PATH is defined as 260 characters, but Windows supports longer path if longer paths support is enabled."